src/components/DataView/example/DataViewKitchenSink.tsx

```diff
@@ -656,7 +656,15 @@ function DataViewKitchenSink(): ReactElement {
 				skip: 0
 			});
 		},
-		onSavedViewRemove: function (data) {
+		onSavedViewRemove: function (data: DataViewProps["savedView"]) {
+			if (data.id === state.savedView.id) {
+				setState({
+					...state,
+					...data.state,
+					savedView: defaultView,

```

Comment:
This changes only the title of the view but doesn't actually change all the filters / configs. Messing around I think one solution could be to just directly call gridConfig.onSavedViewChange(defaultView), that way it changes everything to the defaultView config.